### PR TITLE
Added LZMA2 algorithm support for TarTask

### PR DESCRIPTION
**Pull request by dmromanov**
Since TarTask is based on "pear/Archive_Tar" and the latter is capable of creating lzma2-compressed archives I've exposed an option to create such archives.

### CODE REVIEW

**Comment by mrook** [comment]
Thanks! Can you also add this new option to the user guide?

**Comment by dmromanov** [comment]
Done.

File: docs/docbook5/en/output/chunkhtml/TarTask.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
+                            contents will be included in tar)</td><td>none</td><td>No</td></tr><tr><td><code class="literal">compression</code></td><td><code class="literal">String</code></td><td>Type of compression to use (gzip, bzip2, lzma2, none)</td><td>none</td><td>No</td></tr><tr><td><code class="literal">includeemptydirs</code></td><td><code class="literal">Boolean</code></td><td>If set to <code class="literal">true</code>, also empty directories are copied. </td><td><code class="literal">true</code></td><td>No</td></tr><tr><td><code class="literal">longfile</code></td><td><code class="literal">String</code></td><td>How to handle long files, those with a path &gt; 100 chars. Allowable
```
Comment (mrook) [inline docs/docbook5/en/output/chunkhtml/TarTask.html]: Sorry, should have been more clear! Please add to the source file, `docs/docbook5/en/source/appendixes/optionaltasks.xml`.

Comment (dmromanov) [inline docs/docbook5/en/output/chunkhtml/TarTask.html]: @mrook Should I revert changes in this file?

Comment (mrook) [inline docs/docbook5/en/output/chunkhtml/TarTask.html]: Please, the output is periodically rebuilt anyway :)

Comment (dmromanov) [inline docs/docbook5/en/output/chunkhtml/TarTask.html]: Done.

**Comment by mrook** [comment]
Thanks!

**Pull request merged**